<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>CompletionConfigWidget</class>
 <widget class="QWidget" name="CompletionConfigWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="sorting" >
     <property name="title" >
      <string>Sorting</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkBox" >
        <property name="text" >
         <string>Alphabetical</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_2" >
        <property name="text" >
         <string>Reverse</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_4" >
        <property name="text" >
         <string>Sort based on metadata</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Available:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget_2" />
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Selected:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QToolButton" name="toolButton" >
              <property name="text" >
               <string>&lt;-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_2" >
              <property name="text" >
               <string>-></string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget_3" />
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QToolButton" name="toolButton_3" >
            <property name="text" >
             <string>^</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_4" >
            <property name="text" >
             <string>\/</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="filtering" >
     <property name="title" >
      <string>Filtering</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkBox_3" >
        <property name="text" >
         <string>Hide completions with the following attributes:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget" />
      </item>
      <item>
       <widget class="KIntNumInput" name="kintnuminput" >
        <property name="label" >
         <string>Maximum inheritance depth</string>
        </property>
        <property name="minimum" >
         <number>0</number>
        </property>
        <property name="maximum" >
         <number>20</number>
        </property>
        <property name="specialValueText" >
         <string>Infinity</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grouping" >
     <property name="title" >
      <string>Grouping</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="columnMerging" >
     <property name="title" >
      <string>Column Merging</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="columnMergeTree" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>3</hsizetype>
          <vsizetype>7</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <column>
         <property name="text" >
          <string>Columns</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Merged</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Shown</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QToolButton" name="columnUp" >
          <property name="text" >
           <string>^</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="columnDown" >
          <property name="text" >
           <string>\/</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>105</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
